source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js

```diff
@@ -326,15 +358,20 @@
                             valuesByItem.values.splice(1, 0, averageValue);
 
                             if (self.isCompanyCalificationsEmpty) {
-                                    var companyCalification = _.find(valuesByItem.values, function (calification) {
-                                        return calification.owner == 3;
-                                    });
+                                var companyCalification = _.find(valuesByItem.values, function (calification) {
+                                    return calification.owner == 3;
+                                });
 
                                 if (companyCalification) {
                                     companyCalification.value(averageValue.value());
                                 }
                             }
                         }
+                        valuesByItem.hasComments = ko.computed(function () {
```

Comment:
This is the value that we'll use to show or hide the button that displays the comment section